dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -53,7 +59,32 @@
     private final ProgressBar progressBar = ProgressBar.getInstance();
     private static Map<String, Object> envMap = null;
 
-    public void synthesizeAndDownloadFeatures(List<String> mavenCoords, String dLocation, MavenRepository repository) throws InstallException {
+    final ExecutorService executor = Executors.newCachedThreadPool();
```

Comment:
This ExecutorService should be shutdown when users are done with the ArtifactDownloader object. 

A simple way to do this would be to implement `AutoCloseable`: https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html

Then, when users create the object they can do so within a try-with-resources block or will be expected to run the `close()` method themselves.